app/models/comment.rb

```diff
@@ -0,0 +1,8 @@
+class Comment < ApplicationRecord
+  belongs_to :post
+  belongs_to :user
+
+  def update_post_comments_counter
+    post.update(comments_counter: post.comments.count)
+  end
```

Comment:
- [ OPTIONAL ] I think it would be nice if you use one of the [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) - after_save to call the #update_user_comment_counter method every time a new comment has been saved to database, then make the update_user_comment_counter method private. 💪🏻♻🔥🚀